src/main.rs

```diff
@@ -18,11 +19,11 @@ use gluon::{new_vm, Compiler, Thread, Error, Result};
 use gluon::vm::thread::ThreadInternal;
 #[cfg(not(test))]
 use gluon::vm::Error as VMError;
-#[cfg(not(test))]
-use clap::{Arg, App};
 
 mod repl;
 
+const GLUON_VERSION: &'static str = "0.3.0";
```

Comment:
You could use [CARGO_PKG_VERSION](http://doc.crates.io/environment-variables.html#environment-variables-cargo-sets-for-crates).